docs/advanced-config.md

```diff
@@ -11,13 +11,13 @@ Below you will find all the configuration options supported by this distribution
 
 ## Splunk distribution configuration
 
-| System property                        | Environment variable                   | Default value           | Purpose |
-| -------------------------------------- | -------------------------------------- | ----------------------- | ------- |
-| `splunk.trace-response-header.enabled` | `SPLUNK_TRACE_RESPONSE_HEADER_ENABLED` | `true`                  | Enables adding server trace information to HTTP response headers. See [this document](server-trace-info.md) for more information.
-| `splunk.access.token`                  | `SPLUNK_ACCESS_TOKEN`                  | unset                   | (Optional) Auth token allowing exporters to communicate directly with the Splunk cloud, passed as `X-SF-TOKEN` header. Currently the [both Jaeger and OTLP trace exporters](#trace-exporters) and [SignalFx metrics exporter](metrics.md) support this property.
-| `splunk.metrics.enabled`               | `SPLUNK_METRICS_ENABLED`               | `true`                  | Enables exporting metrics. See [this document](metrics.md) for more information.
-| `splunk.metrics.endpoint`              | `SPLUNK_METRICS_ENDPOINT`              | `http://localhost:9943` | The SignalFx metrics endpoint to connect to.
-| `splunk.metrics.export.interval`       | `SPLUNK_METRICS_EXPORT_INTERVAL`       | `30000`                 | The interval between pushing metrics, in milliseconds.
+| System property                        | Environment variable                   | Default value           | Support | Purpose                                                                                                                                                                                                                                                          |
+| -------------------------------------- | -------------------------------------- | ----------------------- | ------- | -------                                                                                                                                                                                                                                                          |
+| `splunk.access.token`                  | `SPLUNK_ACCESS_TOKEN`                  | unset                   | Beta  | (Optional) Auth token allowing exporters to communicate directly with the Splunk cloud, passed as `X-SF-TOKEN` header. Currently the [both Jaeger and OTLP trace exporters](#trace-exporters) and [SignalFx metrics exporter](metrics.md) support this property.
```

Comment:
Is `splunk.access.token` beta too? I thought that it was a stable feature.

Also, should we use word "beta" or "experimental"? I think we use "experimental" for all other features, the whole project is in "beta".

Reply:
Idea is to mark stable with 1.0 release. Experimental is likely a better term -- will update.